internal/provider/resource_gitlab_project_test.go

```diff
@@ -1620,3 +1876,68 @@ resource "gitlab_project" "foo" {
 }
 	`, rInt, rInt)
 }
+
+func testAccGitlabProjectConfigEE(rInt int) string {
+	return fmt.Sprintf(`
+resource "gitlab_project" "foo" {
+  name = "foo-%d"
+  path = "foo.%d"
+  description = "Terraform acceptance tests"
+  default_branch = "main"
+
+  # NOTE: replaces by topics
+  # tags = [
+  # "tag1",
+  # ]
+
+  # So that acceptance tests can be run in a gitlab organization
+  # with no billing
+  visibility_level = "public"
+  merge_method = "ff"
+  only_allow_merge_if_pipeline_succeeds = true
+  only_allow_merge_if_all_discussions_are_resolved = true
+  squash_option = "default_off"
+  pages_access_level = "public"
+  build_coverage_regex = "foo"
+  allow_merge_on_skipped_pipeline = false
+  ci_config_path = ".gitlab-ci.yml@mynamespace/myproject"
+  resolve_outdated_diff_discussions = true
+  analytics_access_level = "enabled"
+  auto_cancel_pending_pipelines = "enabled"
+  auto_devops_deploy_strategy = "continuous"
+  auto_devops_enabled = true
+  autoclose_referenced_issues = true
+  build_git_strategy = "fetch"
+  build_timeout = 42 * 60
+  builds_access_level = "enabled"
+  container_expiration_policy {
+	enabled = true
+  	cadence = "1month"
+  }
+  container_registry_access_level = "enabled"
+  emails_disabled = true
+  forking_access_level = "enabled"
+  issues_access_level = "enabled"
+  merge_requests_access_level = "enabled"
+  operations_access_level = "enabled"
+  public_builds = false
+  repository_access_level = "enabled"
+  repository_storage = "default"
+  # FIXME: this is GitLab 14.9 only, which isn't released yet. Thus, wait for it before we can use it here ...
+  # security_and_compliance_access_level = "enabled"
+  snippets_access_level = "enabled"
+  topics = ["foo", "bar"]
+  wiki_access_level = "enabled"
+  squash_commit_template = "hello squash"
+  merge_commit_template = "hello merge"
+
+  // EE features
+  approvals_before_merge = 2
+  external_authorization_classification_label = "test"
+  requirements_access_level = "enabled"
+  // are tested in separate test case
+  // mirror_trigger_builds = true
+  // mirror = true
```

Comment:
style: to not mix comment styles...

```suggestion
  # EE features
  approvals_before_merge = 2
  external_authorization_classification_label = "test"
  requirements_access_level = "enabled"
  # are tested in separate test case
  # mirror_trigger_builds = true
  # mirror = true
```